### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by Minaeskandar** [blissend/hqtrivia-automation#17]
Hi I really like all of this but the only downside is I don’t own a Mac...I own a Windows. Is there anyway for me to get this to work on a Windows??

**Comment by ghost** [comment]
I agree all the code would be close to same except for the quicktime part and the pyobjc module too right?

**Comment by Minaeskandar** [comment]
How would I get it to work on a windows?

**Comment by TravelingTrader** [comment]
Any progress on a windows version? Attempting with 
pyscreenshot as Imagegrab
but running into issues integrating after taking the screenshot
